public/index.html

```diff
@@ -2,14 +2,18 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/map-marked-alt-solid.svg" />
```

Comment:
Great, our app has icons! 🎉

(Only I'm a little curious 😄: this 'map-marked-alt-solid.svg' icon, did you create it?  or did you download it somewhere?)

Reply:
In the PR description, @jssckbl says they got it from Font Awesome. I think it works really well for this purpose!

Reply:
Thank you, Alejandro, I've update the QA to add steps for Android.